projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -33,7 +33,13 @@
 		button {
 			line-height: 1.3;
 			box-sizing: border-box;
-			white-space: nowrap;
+			/*
+			 * These next two rules override the default styles for RichText component
+			 * which is used for the button label coming from
+			 * https://github.com/WordPress/gutenberg/blob/b9232527214cd3d523badc4a66b529b7efe615ca/packages/rich-text/src/component/use-default-style.js.
+			 */
+			white-space: nowrap !important;
+			min-width: auto !important;
```

Comment:
I don't think this `min-width` override is necessary. The issue is fixed in my test env without it.